Zeplin.sketchplugin/Contents/Sketch/utils.cocoascript

```diff
@@ -168,6 +168,18 @@ var writeDirectives = function (directives, path) {
     return path;
 }
 
+var getShareId = function(context) {
+    var shareUrl = context.document.cloudShare();
+    if (!shareUrl) {
+        return nil;
+    }
+
+    var urlPartitioned = shareUrl.toString().split("/");
+    var shareId = urlPartitioned[urlPartitioned.length - 1];
+
+    return shareId;
```

Comment:
`context.document.cloudShare()` returns `SCKShare` object ([link](https://github.com/abynim/Sketch-Headers/blob/master/Headers/SCKShare.h#L53)) and it has some properties/methods like `shortID()`. So we can use it to get rid of `urlPartitioned` parts. It would be nice to check the other data in this object to understand collecting more of them is required or not.